app/src/main/java/com/framgia/forder/data/model/DataSuggest.java

```diff
@@ -0,0 +1,67 @@
+package com.framgia.forder.data.model;
+
+import android.os.Parcel;
+import com.arlib.floatingsearchview.suggestions.model.SearchSuggestion;
+import io.realm.RealmObject;
+
+/**
+ * Created by Age on 4/27/2017.
+ */
+
+public class DataSuggest extends RealmObject implements SearchSuggestion {
+    private static boolean mIsHistory = false;
+    private String mName;
+    private int mDomainId;
+
+    public DataSuggest() {
+    }
+
+    public DataSuggest(int domainId, String name) {
+        mDomainId = domainId;
+        mName = name;
+    }
+
+    public DataSuggest(String name) {
+        mName = name;
+    }
+
+    public static boolean getIsHistory() {
```

Comment:
WARNING: [BooleanGetMethodName] A getX() method which returns a boolean should be named isX()
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#BooleanGetMethodName